### ISSUE TITLE
Error while hiding paths

### ISSUE THREAD

**Issue by samgozman** [meerk40t/meerk40t#182]
Steps to reproduce:
1) Insert any SVG file
2) Click "Hide paths"
3) Enjoy 😊 

```
MeerK40t crash log. Version: 0.5.2, Windows 10 x64
Traceback (most recent call last):
  File "site-packages\wx\core.py", line 3285, in <lambda>
  File "wxMeerK40t.py", line 700, in refresh_in_ui
  File "wxMeerK40t.py", line 716, in update_buffer_ui_thread
  File "wxMeerK40t.py", line 1168, in on_draw_scene
  File "LaserRender.py", line 46, in render
  File "LaserRender.py", line 46, in <listcomp>
  File "Kernel.py", line 702, in __getitem__
TypeError: ConfigBase.Read(): argument 1 has unexpected type 'int'

```

**Comment by tatarize** [comment]
Corrected in 0.6.0

**Comment by tatarize** [comment]
This issue was corrected in 0.6.0. Which is now in full release.
